extensions/qute/deployment/src/test/java/io/quarkus/qute/deployment/typesafe/Item.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.qute.deployment.typesafe;
+
+import java.util.Set;
+
+public class Item {
+
+    String name;
+
+    OtherItem[] otherItems;
+
+    public Item(String name, Set<OtherItem> otherItems) {
```

Comment:
Just nitpicking... but why not `OtherItem... otherItems`?

Reply:
I'm used to Sets rather than arrays. I pushed version with varargs as it looks better.

Reply:
> I'm used to Sets rather than arrays.

Sure. It's just a little bit easier to use (and less allocations ;-).